### PR TITLE
[docs] - cesium/atom - minor reviews to documentation - indentation, text layout, and so on

### PR DESCRIPTION
**Pull request by tem44**
#### [docs] - cesium/atom - minor reviews to documentation - indentation, text layout, and so on

### CODE REVIEW

**Comment by ZePedroResende** [comment]
In my opinion this decreases the readbility of the ReadMe. 
@0urobor0s thoughs ?

File: README.md [hunk 1]
```diff
@@ -10,44 +10,31 @@ This project is the CeSIUM management platform.
...
 
-
 ### Members
 
 - Profile edition
 - Attend in events
 - Consult history of events attendance
 - Check other members profile
 
+---
```
Comment (0urobor0s) [inline README.md]: Why?

File: README.md [hunk 2]
```diff
@@ -10,44 +10,31 @@ This project is the CeSIUM management platform.
...
+
```
Comment (0urobor0s) [inline README.md]: Again, why?

File: README.md [hunk 3]
```diff
@@ -10,44 +10,31 @@ This project is the CeSIUM management platform.
...
 ### Production
 
-If you wish to deploy the master branch to production use:
-
-`bin/deploy production`
-
-If, for some reason, you wish to deploy another branch other than master to production use:
-
-`bin/deploy production [BRANCH]`
+If you wish to deploy the master branch to production use: `bin/deploy production` If, for some reason, you wish to deploy another branch other than master to production use: `bin/deploy production [BRANCH]`
```
Comment (0urobor0s) [inline README.md]: Did you check the generated file?

File: README.md [hunk 4]
```diff
@@ -10,44 +10,31 @@ This project is the CeSIUM management platform.
...
 
 ### Staging
 
-To deploy master to staging use:
+To deploy master to staging use: `bin/deploy`. To deploy any other branch to staging use: `bin/deploy staging [BRANCH]`
```
Comment (0urobor0s) [inline README.md]: Again.

File: README.md [hunk 5]
```diff
@@ -10,44 +10,31 @@ This project is the CeSIUM management platform.
...
 
-`bin/deploy`
-
-To deploy any other branch to staging use:
-
-`bin/deploy staging [BRANCH]`
+---
```
Comment (0urobor0s) [inline README.md]: Why?

File: README.md [hunk 6]
```diff
@@ -10,44 +10,31 @@ This project is the CeSIUM management platform.
...
-To run a rails console on the staging app run:
-
-`bin/console`
+We use the [capistrano-rails-console](https://github.com/ydkn/capistrano-rails-console) gem to be able to run a remote rails console on the web app. To run a rails console on the staging app run: `bin/console` If you wish to run a rails console on the production app run: `bin/console production`
 
-If you wish to run a rails console on the production app run:
+---
 
-`bin/console production`
+<img src="https://avatars.githubusercontent.com/u/4223" width="45" height="auto"></img><img src="http://cesium.di.uminho.pt/assets/logo-130aa2b71d4ca3daf2922a30af8574d2356002f2ac0e8059ff529754b0d935bd.png" width="120" height="auto"></img>
```
Comment (0urobor0s) [inline README.md]: Should never use a non static image. You should readme assets to repository.

File: README.md [hunk 7]
```diff
@@ -10,44 +10,31 @@ This project is the CeSIUM management platform.
...
 
-We use the [capistrano-rails-console](https://github.com/ydkn/capistrano-rails-console) gem to be able to run a remote rails console on the web app.
-
-To run a rails console on the staging app run:
-
-`bin/console`
+We use the [capistrano-rails-console](https://github.com/ydkn/capistrano-rails-console) gem to be able to run a remote rails console on the web app. To run a rails console on the staging app run: `bin/console` If you wish to run a rails console on the production app run: `bin/console production`
 
-If you wish to run a rails console on the production app run:
+---
```
Comment (0urobor0s) [inline README.md]: Why?

**Pull request closed without merging**